test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/disposaldetails/DisposalDetailsControllerSpec.scala

```diff
@@ -3709,20 +3709,21 @@ object DisposalDetailsControllerSpec extends Matchers {
 
   def validateDisposalDetailsCheckYourAnswersPage(
     disposalDetailsAnswers: CompleteDisposalDetailsAnswers,
-    doc: Document
+    doc: Document,
+    isIndirectDisposal: Boolean = false
   ): Unit = {
-    doc
-      .select("#propertyShare-answer")
-      .text()
-      .stripSuffix(
-        "%"
-      )       shouldBe disposalDetailsAnswers.shareOfProperty.percentageValue
-      .toString()
+    if (!isIndirectDisposal)
+      doc
+        .select("#propertyShare-answer")
+        .text()
+        .stripSuffix("%") shouldBe disposalDetailsAnswers.shareOfProperty.percentageValue.toString()
```

Comment:
to really check that it isn't displayed for indirect disposals you could assert something like
```scala
doc.select("#propertyShare-answer").text() shouldBe ""
```

Reply:
Yes I forgot that one. nice one